.github/workflows/sonar-qube-scann.yml

```diff
@@ -1,34 +1,64 @@
-#SonarQube Configuration
-# This is the sonarqube configuration, check readme for instructions
-#name: 'sonarqube'
-#
-#on: push
-#
-#jobs:
-#  sonarQubeTrigger:
-#    name: Sonarqube-Trigger
-#    runs-on: ubuntu-latest
-#    steps:
-#      - uses: dart-lang/setup-dart@v1
-#      - name: Checkout code
-#        uses: actions/checkout@v2
-#      - uses: webfactory/ssh-agent@v0.9.0
-#        with:
-#          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
-#      - name: Set up Flutter
-#        uses: subosito/flutter-action@v2
-#        with:
-#          channel: stable
-#          flutter-version: 3.24.3
-#      - run: flutter --version
-#      - name: Get Dependencies
-#        run: flutter pub get app && flutter pub get modules/domain && flutter pub get modules/data && flutter pub get modules/common
-#      - name: Analyze App
-#        #run analyze first
-#        run: flutter analyze
-#      - name: Setup Sonarqube Scanner
-#        uses: warchant/setup-sonar-scanner@v8
-#      - name: Run Sonarqube Scanner
-#        run: sonar-scanner
-#          -Dsonar.token=${{ secrets.SONAR_TOKEN }}
-#          -Dsonar.host.url=${{ secrets.SONAR_URL }}
+# name: sonarqube
+
+# # ────────────────────────────────────────────────────────────────
+# # CI TRIGGERS
+# #   · push on main      → historical baseline
+# #   · pull_request PRs  → quality gate before merge
+# # ────────────────────────────────────────────────────────────────
+# on:
+#   push:
+#     branches: [main]
+#   pull_request:
+#     types: [opened, synchronize, reopened]
+
+# jobs:
+#   sonarQubeTrigger:
+#     name: Sonarqube Trigger
+#     runs-on: ubuntu-latest
+
+#     steps:
+#       # 1 — Checkout the repo
```

Comment:
I hitnk you are missing the python instalation
- name: Install Python dependencies
        run: |
          python3 -m pip install --upgrade pip
          # Add any required pip install commands here, e.g.:
          # pip install somepackage